lib/internal/child_process.js

```diff
@@ -610,9 +610,6 @@ function setupChannel(target, channel) {
         this.emit('error', ex);  // FIXME(bnoordhuis) Defer to next tick.
       }
     }
-
-    /* If the master is > 2 read() calls behind, please stop sending. */
-    return channel.writeQueueSize < (65536 * 2);
```

Comment:
This is not a good change.  I remember this was introduced deliberately because unbound queue growth was a real problem with some applications.